targets/TARGET_Realtek/TARGET_AMEBA/ota_api.h

```diff
@@ -22,17 +22,20 @@
 #define FLASH_SECTOR_SIZE    0x1000
 #define FLASH_SECTOR_MASK    ~(FLASH_SECTOR_SIZE - 1)
 
+#define OTA_REGION1_HEADER  0x0b000
+#define OTA_REGION2_HEADER  0x0c000
+#define OTA_REGION2_BASE    0x120000
 #define OTA_REGION1_BASE    0x40000
```

Comment:
Reorder OTA_REGION1_BASE and OTA_REGION2_BASE defines ?

Reply:
Sure, will do. Thank you @tung7970 . BTW, how can I re-open this PR? Do I need to raise a new one?

Reply:
There should be a 'Reopen pull request' at the bottom of the page.

Reply:
It seems that I don't have right to re-open it. I created a new one #5858, please help review it, thanks. @0xc0170 @tung7970